biogeochem/FatesAllometryMod.F90

```diff
@@ -305,7 +306,7 @@ subroutine h2d_allom(h,ipft,d,dddh)
                 p3          => prt_params%allom_d2h3(ipft), &
                 allom_hmode => prt_params%allom_hmode(ipft))
 
-      select case(int(allom_hmode))
```

Comment:
agreed, smarter to convert the switch on the read in, no need to convert over and over again in the dynamics loop